### PR TITLE
Improve lidar doc

### PR DESCRIPTION
**Pull request by DavidMansolino**
Linked with omichel/webots#3775

### CODE REVIEW

**Comment by DavidMansolino** [comment]
:+1:

File: reference/lidar.md [hunk 1]
```diff
@@ -170,16 +170,17 @@ bool wb_lidar_is_point_cloud_enabled(WbDeviceTag tag)
...
+`wb_lidar_enable_point_cloud()` allows the user to enable the lidar point cloud update, the point cloud array is then updated with the same sampling period as the range image.
 
 `wb_lidar_disable_point_cloud()` allows the user to disable the lidar point
 cloud update.
 
 The `wb_lidar_is_point_cloud_enabled()` function returns true if the point cloud
 update is enabled or false otherwise.
 
+> **note**:
+To get the point cloud array, enabling the point cloud is not sufficient, first the lidar should be enabled using the `wb_lidar_enable()` function.
```
Comment (stefaniapedrazzi) [inline reference/lidar.md]: Personally I would use a '.' instead of ',' before _first_,  but it is also ok like this.

**Comment by stefaniapedrazzi** [comment]
:+1:

**Comment by DavidMansolino** [comment]
:+1:

**Comment by stefaniapedrazzi** [comment]
:+1:

**Pull request merged**